extension/goext/schemas.go

```diff
@@ -57,6 +58,49 @@ type Paginator struct {
 	Offset uint64
 }
 
+// Below code is adapted from similar code in db/pagination/pagination.go, but
+// doesn't validate parameters.
+type OptionPaginator func(*Paginator)
```

Comment:
Unfortunately there is bit of duplication here, is it possible to convert `paginator` into an interface?